### DISCUSSION TITLE
Apple TV app?

### DISCUSSION THREAD

**Discussion by ghost** [qnblackcat/uYouPlus#440] [Ideas]
I don't know how different the AppleTv app is from the one you guys work on but that would be SO useful, the ads are so much more obnoxious over there.

Thank you

**Comment by tom12011** [comment]
I'd love that. There aren't many tvOS app alternatives and they are either very laggy or just ugly.  The ads on apple devices are just unrelenting.

**Comment by qnblackcat** [comment]
No, and probably never
